README.md

```diff
@@ -264,6 +264,32 @@ colord("#00336680").toCmykString(); // "device-cmyk(100% 50% 0% 60% / 0.5)"
 
 </details>
 
+<details>
+  <summary><code>Harmonies</code></b> (<b>Harmony colors</b> plugin)</summary>
+
+Provides functionatity to generate [harmony colors](https://en.wikipedia.org/wiki/Harmony_(color)).
+
+```js
+import { colord, extends } from "colord";
+import { harmonies } from "colord/plugins/harmonies";
+
+const color = colord("FF0000");
+
+color.harmonies("analogous")
+  .map(color => color.hex); // [ "#FF0080", "#FF0000", "#FF8000"]
+color.harmonies("complimentary")
+  .map(color => color.hex); // [ "#FF0000", "#00FFFF" ]
+color.harmonies("rectangle")
+  .map(color => color.hex); // [ "#FF0000", "#FFFF00", "#00FFFF", "#0000FF" ]
+color.harmonies("tetradic")
+  .map(color => color.hex); // [ "#FF0000", "#80FF00", "#00FFFF", "#8000FF" ]
+color.harmonies("triadic"  )
+  .map(color => color.hex); // [ "#FF0000", "#00FF00", "#0000FF" ]
+color.harmonies("splitcomplimentary")
```

Comment:
Don't you think we should call it split-complimentary or splitComplimentary?

Reply:
I think because it is string, we may use kebab case. I think it is more readable. But it is up to you, what do you like?